ledger/ledger-api-client/src/test/suite/scala/com/digitalasset/ledger/client/services/commands/CommandTrackerFlowTest.scala

```diff
@@ -453,27 +475,79 @@ class CommandTrackerFlowTest
       }
     }
 
+    "a completion without submission id arrives" should {
```

Comment:
```suggestion
    "an incoming completion without submission id" should {
```

Reply:
I think this doesn't work with the current naming.

It would be:
`Command tracking flow when an incoming completion without submission id should...`
vs existing:
`Command tracking flow when a completion without the submission id arrives should...`